client/src/Client.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+
+function checkStatus(response) {
+  if (response.status >= 200 && response.status < 300) {
+    return response;
+  } else {
+    const error = new Error(`${response.statusText}`);
+    error.status = response.statusText;
+    error.response = response;
+    throw error;
+  }
+}
+
+function parseJSON(response) {
+  return response.json();
+}
+
+class Client {
```

Comment:
It is better to separate the code that manipulates/retrieves articles and users data.

Reply:
Done

Reply:
Done